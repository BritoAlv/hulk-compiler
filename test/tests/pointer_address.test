print(print(2)); # Prints pointer address