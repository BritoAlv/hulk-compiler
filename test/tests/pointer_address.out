2.00000000
268576536

